Content.Server/Construction/Conditions/AmeShieldIntegrity.cs

```diff
@@ -0,0 +1,43 @@
+using Content.Server.Ame.Components;
+using Content.Shared.Construction;
+using JetBrains.Annotations;
+using Content.Shared.Examine;
+
+namespace Content.Server.Construction.Conditions;
+
+    [UsedImplicitly]
+    [DataDefinition]
+    public sealed partial class AmeShieldIntegrity : IGraphCondition
+    {
```

Comment:
Are we supposed to indent classes if we are using file-scoped namespaces?

Reply:
Hell naw